A38-admin-interface-api.md

```diff
@@ -3,34 +3,29 @@ Admin Interface API
 * Author(s): lidiz
 * Approver: markdroth
 * Status: In-Review
-* Implemented in: C++, Golang, Java
+* Implemented in: all languages
```

Comment:
This should not list languages that it is not actually implemented in yet.  For now, leave it empty; we can populate it later as it gets implemented in each language.

Reply:
Removed.

Makes me wonder why didn't we set a `Planned in` field, if it is a proposal.